lib/reporters/mini.js

```diff
@@ -163,6 +164,21 @@ class MiniReporter {
 				}
 
 				status += '  ' + colors.title(test.title) + '\n';
+
+				if (test.logs) {
+					test.logs.forEach(log => {
+						const logLines = indentString(colors.log(log), 6);
+						const logLinesWithLeadingFigure = logLines.replace(
+							/^ {6}/,
+							`    ${colors.information(figures.info)} `
+						);
```

Comment:
@novemberborn the reason I didn't do that was I only wanted to `i` symbol to show up once per log line (to indicate where a log message starts)

```
    * this is a message that will have multiple lines, but there only the first line
      will have the `*` to indicate a log message, and the indent of the other lines
      is at the indent level of the first log message (after the `*`)
    * Some other log message
```

Does that make sense? Or would you rather it be

```
    * this is a message that will have multiple lines, but there only the first line
    * will have the `*` to indicate a log message, and the indent of the other lines
    * is at the indent level of the first log message (after the `*`)
    * Some other log message, but how do I know that.
```